app/v2/notifications/notification_schemas.py

```diff
@@ -123,7 +123,6 @@
         },
         "is_csv": {
             "format": "send_a_file_is_csv",
-            "required": False,
```

Comment:
How come we want to remove this? We have required False on the other ones? Maybe we should be keeping this and also adding required False to `filename` too?

Reply:
`required: False` is the default. There are required statements further down now for both `is_csv` and `filename`, so I thought having them just there might be clearer. Otherwise they say `required: false` here, but then further down there's another (inverted) `require` on them.

I think having `required: false` here looks like it's doing something independent, but the rules around this parameter are now tightly bound to the rules around `filename`, so I thought co-locating them might be clearer...

JSON Schema requireds are a bit weird though so I'm not sure what's best. Happy to put them back here.

Reply:
thanks for explaining. Pros and cons either way. Pretty minor impact either way.

Your choice! I'm happy either way.